src/dmd/backend/blockopt.d

```diff
@@ -51,12 +57,17 @@ else
 
 extern(C++):
 
-extern(C) void *mem_fcalloc(size_t numbytes); // tk/mem.c
-void *mem_free(size_t numbytes); // tk/mem.c
+extern (C) void *mem_fcalloc(size_t numbytes); // tk/mem.c
+extern (C) void *mem_free(void*); // tk/mem.c
```

Comment:
The Visual Studio build doesn't like this:

```
..\dmd\backend\blockopt.d(61): error : Function type does not match previously declared function with the same mangled name: `mem_free` [C:\projects\dmd\src\vcbuild\dmd.vcxproj]
Done Building Project "C:\projects\dmd\src\vcbuild\dmd.vcxproj" (default targets) -- FAILED.
Done Building Project "C:\projects\dmd\src\vcbuild\dmd.vcxproj.metaproj" (default targets) -- FAILED.
Done Building Project "C:\projects\dmd\src\vcbuild\dmd.sln" (dmd target(s)) -- FAILED.
Build FAILED.
"C:\projects\dmd\src\vcbuild\dmd.sln" (dmd target) (1) ->
```

Reply:
The only other definitions are:

```
src/dmd/backend/go.d
93:extern (C) void mem_free(void* p);

src/dmd/backend/memh.d
26:void mem_free(void *);
```

Reply:
It shouldn't return a `void*`